duckduckgo/Android
# Add pixels for autofill

Task/Issue URL: https://app.asana.com/0/0/1203013158998989/f

### Description
Adds pixels around autofill usage, including a one-time pixel around whether the device is autofill-capable (meaning secure storage available and device auth enabled).

### Steps to test this PR
See the list detailed in [Identify autofill pixels required](https://app.asana.com/0/1203013158998988/1203013158998988/f); run through the autofill scenarios and ensure the pixels are what is expected.
